packages/framework-core/src/booster-read-models-reader.ts

```diff
@@ -18,9 +20,39 @@ import { getReadModelFilters } from './services/filter-helpers'
 export class BoosterReadModelsReader {
   public constructor(readonly config: BoosterConfig, readonly logger: Logger) {}
 
-  public async fetch(readModelRequest: ReadModelRequestEnvelope): Promise<Array<ReadModelInterface>> {
+  public async findById(readModelByIdRequestEnvelope: ReadModelByIdRequestEnvelope): Promise<ReadModelInterface> {
+    this.validateByIdRequest(readModelByIdRequestEnvelope)
+    const readModelMetadata = this.config.readModels[readModelByIdRequestEnvelope.typeName]
+    const searcher = Booster.readModel(readModelMetadata.class)
+
+    const filters = getReadModelFilters({}, readModelMetadata.before, readModelByIdRequestEnvelope.currentUser)
+
+    searcher.filter(filters)
+
+    return searcher.findById(readModelByIdRequestEnvelope.id, readModelByIdRequestEnvelope.sequenceKey)
+  }
+
+  public async search(
+    readModelRequest: ReadModelRequestEnvelope
+  ): Promise<ReadModelInterface[] | ReadModelListResult<ReadModelInterface>> {
```

Comment:
Nitpick, usually we use the following form of array type throughout the codebase:
```suggestion
  ): Promise<Array<ReadModelInterface> | ReadModelListResult<ReadModelInterface>> {
```